app/models/concerns/nc_residence_county_concern.rb

```diff
@@ -0,0 +1,53 @@
+# Handles the bits of the North Carolina state file intake that deals with the
+# county.
+module NcResidenceCountyConcern
+  extend ActiveSupport::Concern
+
+  COUNTIES = {
+    # This first one is unofficial and for our uses
+    "unfilled" => "Unfilled",
```

Comment:
I chose this specifically because it is not a number. I didn't want to have an invalid `000` suddenly become valid and this makes that chance more remote.